lib/devise/models/authenticatable.rb

```diff
@@ -187,8 +191,9 @@ def devise_mailer
       def send_devise_notification(notification, *args)
         message = devise_mailer.send(notification, self, *args)
         # Remove once we move to Rails 4.2+ only.
+
         if message.respond_to?(:deliver_now)
```

Comment:
@lucasmazza , but if remove this `if`, that meanings if user's environment don't support `deliver_now` or `deliver_later` will failed.

And really sorry about that I respond so lately. 😭